polkadot-protocol/protocol-components/transactions/transactions-and-block-basis.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Transactions and Block Basis
+description: Overview of transactions and blocks in Polkadot SDK-Based Chains, including signed, unsigned, and inherent transactions.
+---
+
+# Transactions and Block Basis
+
+## Introduction
+
+In this article, you'll learn about the different types of transactions you can create and how to use them in a runtime. Transactions determine the data that makes its way into the blocks in your blockchain. By learning how different transaction types are used, you'll be better prepared to select the appropriate type for your needs.
+
+## What is a Transaction?
+
+In general, transactions provide a mechanism for making changes to a state that can be included in a block. There are three distinct transaction types in Substrate:
```

Comment:
Should we use Polkadot SDK or Substrate here? TBH, I am ok with using Substrate, as by now it should be obvious

Reply:
Let's stick to Substrate for now then, I agree :)